bin/dwi2response

```diff
@@ -62,7 +62,7 @@ def execute(): #pylint: disable=unused-variable
   # Sanitise some inputs, and get ready for data import
   if app.ARGS.lmax:
     try:
-      lmax = [ int(x) for x in app.ARGS.lmax.split(',') ]
+      lmax = app.ARGS.lmax
```

Comment:
May be multiple instances of unnecessary try clauses like this because the type conversion is no longer happening